loopd/start.go

```diff
@@ -99,9 +100,18 @@ func Start() error {
 	// Print the version before executing either primary directive.
 	log.Infof("Version: %v", loop.Version())
 
+	lisCfg := listenerCfg{
```

Comment:
Would be nice to extract this into a helper function as this gets quite large.

Reply:
good idea, done! https://github.com/lightninglabs/loop/pull/115/commits/74eac26336f3d18ef857762901ed518024393f55